custom_functions/fast_etl.py

```diff
@@ -498,12 +498,30 @@ def _build_increm_filter(col_list: list, dest_hook: MsSqlHook,
     return where_condition
 
 
-# New func: Distingue se a coluna para obtenção do max() e montagem do
-#           filtro (where) é uma "data/hora alteração" ou é um número
-#           sequencial (id, pk, etc): 'date_column' ou 'key_column'.
+
 def _build_filter_condition(dest_hook: MsSqlHook,
-                table: str, date_column: str, key_column: str):
+                table: str, date_column: str, key_column: str) -> str:
```

Comment:
```suggestion
                table: str, date_column: str, key_column: str) -> Tuple[str, str]:
```

Na verdade, [o retorno é uma tupla](https://github.com/economiagovbr/FastETL/blob/b781f583166f25286745ab8910ae120f70e3efda/custom_functions/fast_etl.py#L544).